Dockerfile

```diff
@@ -0,0 +1,83 @@
+FROM harbor.k8s.libraries.psu.edu/library/ruby-2.6.6-node-15:20210507 as base
+ARG UID=2000
+
+USER root
+RUN apt-get update && \
+   apt-get install --no-install-recommends -y \
+   sqlite3 \
+   vim \
+   default-libmysqlclient-dev \
+   libsqlite3-dev \
+   shared-mime-info && \
```

Comment:
@whereismyjetpack and I figured out `vim` doesn't need to be there. What about `sqlite3` and the others?

Reply:
SQLite needs to be in the dev stage(for now)  thanks for shining a light on this, I think at one point they were in the right stages and some copy pasta happened. 


Mmmm : 🍝

Reply:
for some reason the bundle won't install without sql dev packages even if we only install the production group. I'd like to, as part of another PR hook up mysql into the docker-compose setup and remove the dep on sqlite so that the local more closely matches prod anyway

Reply:
that's really weird. We don't have any of those in the scholarsphere Dockerfile, but that's using postgres.

Reply:
the Gemfile has in it sqlite3, i think bundle downloads gems even if they aren't installed to do the dependency resolution or something, i think eventually they will be pruned out of there. 

"While the --without option will skip installing the gems in the specified groups, it will still download those gems and use them to resolve the dependencies of every gem in your Gemfile(5)."


https://bundler.io/v2.2/man/bundle-install.1.html

Reply:
what about https://github.com/psu-libraries/scholarsphere/blob/develop/Dockerfile#L41

Reply:
yeah, i tried that too -- same results so i left it

Reply:
I'm a liar. i gave it a csv. i will try that

Reply:
that was it, moved sql* packages down to the dev stage